bin/run_epix

```diff
@@ -40,7 +39,7 @@ def pars_args():
     parser.add_argument('--MaxDelay', dest='max_delay', type=float,
                         action='store', default=1e7, #ns
                         help='Maximal time delay to first interaction which will be stored [ns]')
-    parser.add_argument('--EventRate', dest='event_rate', type=float,
+    parser.add_argument('--EventRate', dest='event_rate_epix', type=float,
```

Comment:
Same argumentation as for `EpixConfigOverride` regarding epix in the name.

Reply:
This is the reason: https://github.com/XENONnT/WFSim/search?q=event_rate.
But I agree we can rename to `source_rate` or something distinctive without the need for `epix` on it.

Reply:
But this is in principle the same parameter or? In both cases the event rate in Hz and if one uses WFsim in combination with EPIX it will also do the same thing.

Reply:
The default event_rate in epix is -1. This will crash wfsim. And it's usage is slightly different. I think it's a bad idea to inherent the value from epix in wfsim (which you have to do if they have the same name). And setting two different default values is not allowed

Reply:
Since we're here, a dummy question: what's the role played by `event_rate` in wfsim, outside of the rand instructions generation?

Reply:
None, but it will complain

Reply:
In that case I suggest we rename this one to `source_rate`, if @HenningSE agrees. And maybe rethink also the wfsim name to something more specific, given its limited usage.